src/parser/template/output.ts

```diff
@@ -1,11 +1,13 @@
-import { assertField, assertString } from '../private/types';
+import { CfnElement } from 'aws-cdk-lib';
+import { Construct } from 'constructs';
+import { assertString } from '../private/types';
 import { schema } from '../schema';
 import { ifField, parseExpression, TemplateExpression } from './expression';
 
 export interface TemplateOutput {
   readonly description?: string;
   readonly value: TemplateExpression;
-  readonly exportName?: TemplateExpression;
+  readonly exportName?: string;
```

Comment:
I read the docs in a way that Intrinsic functions are allowed here.
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/outputs-section-structure.html#outputs-section-structure-examples
> You can use intrinsic functions to customize the Name value of an export. The following examples use the Fn::Join function.

Reply:
Good catch! I missed that section.